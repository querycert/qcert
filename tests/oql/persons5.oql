/* SFW with count */
select m
from m in Marketers
where count(m->products) >= 3
