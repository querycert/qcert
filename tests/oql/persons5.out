[{"$class": ["ProductRep"], "$data": {"clients": ["100", "101"], "id": "128", "name": "James Do", "products": ["2", "4", "5"]}}]